### ISSUE TITLE
oss-fuzz build_fuzzers failed

### ISSUE THREAD

**Issue by henices** [ImageMagick/ImageMagick#935]
python infra/helper.py build_fuzzers --sanitizer address imagemagick

```
+ clang++ -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libc++ -std=c++11 -I/work/include/ImageMagick-7 /src/imagemagick/Magick++/fuzz/enhance_fuzzer.cc -o /out/enhance_fuzzer -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16 -lFuzzingEngine /work/lib/libMagick++-7.Q16HDRI.a /work/lib/libMagickWand-7.Q16HDRI.a /work/lib/libMagickCore-7.Q16HDRI.a                                                                                    
/src/imagemagick/Magick++/fuzz/enhance_fuzzer.cc:7:11: error: no member named 'ResourceLimits' in namespace 'Magick'                                                                          
  Magick::ResourceLimits::memory(1500000000);  
  ~~~~~~~~^                                    
1 error generated.                             
fuzzers build failed.  
```

**Comment by dlemstra** [comment]
Will fix this later today. And I will also try to move the build script to our repository so we can run the build on travis.

**Comment by henices** [comment]
Good news, I will submit code after that has been done.

**Comment by dlemstra** [comment]
Build is fixed in #936
